x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/form_errors.tsx

```diff
@@ -3,36 +3,3 @@
  * or more contributor license agreements. Licensed under the Elastic License;
```

Comment:
Can we delete this file?

Reply:
thank you! that was left after a merge conflict :D